### PR TITLE
edits to changelog for 24.3.0

### PR DESCRIPTION
**Pull request by macfarla**
* correct release number for 24.3.0
* added link to release page (not individual download links)
* correction to X_SNAP X_CHECKPOINT removal target version
* move blob RPC changes into 24.3.0
* BFT (not PoA) won't start with SNAP or CHECKPOINT

### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Most advanced CI tests are deferred until PR approval, but you could:

- [ ] locally run all unit tests via: `./gradlew build`
- [ ] locally run all acceptance tests via: `./gradlew acceptanceTest`
- [ ] locally run all integration tests via: `./gradlew integrationTest`
- [ ] locally run all reference tests via: `./gradlew ethereum:referenceTests:referenceTests`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,9 +39,11 @@
 - `--Xfilter-on-enr-fork-id` has been removed. To disable the feature use `--filter-on-enr-fork-id=false`.
 - `--engine-jwt-enabled` has been removed. Use `--engine-jwt-disabled` instead. [#6491](https://github.com/hyperledger/besu/pull/6491)
 - Release docker images now provided at ghcr.io instead of dockerhub
+- Add blob transaction support to `eth_call` [#6661](https://github.com/hyperledger/besu/pull/6661)
+- Add blobs to `eth_feeHistory` [#6679](https://github.com/hyperledger/besu/pull/6679)
```
Comment (siladu) [inline CHANGELOG.md:43]: Should these be under breaking changes?

Comment (macfarla) [reply CHANGELOG.md:43]: yes indeed

Comment (macfarla) [reply CHANGELOG.md:43]: fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -84,6 +84,7 @@
 - Fix traces so that call gas costing in traces matches other clients traces [#6525](https://github.com/hyperledger/besu/pull/6525)
 
 ### Download Links
+https://github.com/hyperledger/besu/releases/tag/24.3.0
```
Comment (siladu) [inline CHANGELOG.md:87]: We normally add the direct links and SHAs here...is there a reason not to here?

Comment (macfarla) [reply CHANGELOG.md:87]: well on the one hand I was being lazy... on the other hand it's automated in the release creation, it seemed manual to copy it in here. it's only one click away. but there's no version control on the release notes over there so I guess it makes sense to have it here as well

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**